Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Generic/PlantEnable.mo

```diff
@@ -0,0 +1,300 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Generic;
+model PlantEnable
+  "Sequence to enable/disable boiler plant based on heating hot-water requirements"
+
+  parameter Integer nIgnReq(final min=0) = 0
+  "Number of hot-water requests to be ignored before enablng boiler plant loop";
+
+  parameter Integer nSchRow(final min=1) = 4
+  "Number of rows to be created for plant schedule table";
+
+  parameter Real TLocOut(final unit="K",
+                         final displayUnit="degC") = 299.67
+  "Boiler lock-out temperature for outdoor air";
+
+  parameter Real plaOffThrTim(final unit="s",
+                              final displayUnit="s") = 15*60
+  "Minimum time for which the plant has to stay off once it has been disabled";
+
+  parameter Real plaOnThrTim(final unit="s",
+                             final displayUnit="s") = plaOffThrTim
+  "Minimum time for which the boiler plant has to stay on once it has been enabled";
+
+  parameter Real schTab[nSchRow,2] = [0,1; 6,1; 18,1; 24,1]
```

Comment:
@milicag : Do you mean I should have the schTab parameter at the end of list of parameters? Or should the parameter nSchRow be moved next to it?

Reply:
I was thinking of having it at the end of the list, but it is a good point to keep it right after the parameter that defines its dimension. So the rule would be to prioritize the variable type (bool, int, real), then actual function, then dimensionality. How does that feel to you?

Reply:
Ah ok, got it! Yeah that makes sense. It would make it easier for the user to understand what they are modifying by having the array parameter right after the integer dimension parameter